.github/workflows/release.yml

```diff
@@ -323,7 +323,7 @@ jobs:
           srtool --version
       - name: Build Deterministic WASM
         run: |
-          RUST_LOG=debug SRTOOL_TAG="1.70.0" srtool build \
+          RUST_LOG=debug SRTOOL_TAG="1.75.0" srtool build \
```

Comment:
should we do 1.76 here?

Reply:
1.75.0 appears to be the latest version.